packages/react-components/src/components/Select/Select.css

```diff
@@ -48,7 +59,9 @@
 }
 .bcds-react-aria-Select--Button[data-focused] {
   border-color: var(--surface-color-border-active);
-  outline: none;
+  outline: solid var(--layout-border-width-medium)
```

Comment:
This looks great! 👍 👍 

<img width="219" alt="Select component focused with popover open" src="https://github.com/user-attachments/assets/59afe564-5b15-49d1-bfff-ef1594f9a1ab">